erpnext/patches/v14_0/update_posting_datetime.py

```diff
@@ -0,0 +1,14 @@
+import frappe
+
+
+def execute():
+	frappe.reload_doc("Stock", "doctype", "Stock Ledger Entry")
+
+	frappe.db.sql(
+		"""
+		UPDATE `tabStock Ledger Entry`
+			SET posting_datetime = timestamp(posting_date, posting_time)
+	"""
+	)
+
+	frappe.db.add_index("Stock Ledger Entry", ["posting_datetime", "creation"])
```

Comment:
This isn't required. We do need to drop old indexes though.

Reply:
Fixed